homeassistant/components/light/hue.py

```diff
@@ -211,7 +211,7 @@ def process_groups(hass, api, bridge, bridge_type, update_lights_cb):
 
         if lightgroup_id not in groups:
             groups[lightgroup_id] = HueLight(
-                int(lightgroup_id), info, bridge,
+                hass, int(lightgroup_id), info, bridge,
```

Comment:
This is the wrong fix. See my comment in the linked issue. Avoid the race via eg our dispatch helper and `async_added_to_hass`. You can look at the mysensors component for an example.